easybuild/easyconfigs/a/ABINIT/ABINIT-7.6.2_named-constant.patch

```diff
@@ -0,0 +1,16 @@
+--- src/62_ctqmc/defs.h.orig	2014-02-15 21:30:20.000000000 +0000
```

Comment:
@AFCJamie please include comments above on what this patch does, and where you got it from (any lines above this line are comments)